Update doc for ShareWithTeam cmdlets (#9842)
MicrosoftDocs/office-docs-powershell: teams/teams-ps/teams/Get-SharedWithTeamUser.md

---
external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
Module Name: MicrosoftTeams
online version: https://docs.microsoft.com/powershell/module/teams/get-sharedwithteamuser
schema: 2.0.0
author: zhongxl
ms.author: zhongxl
ms.reviewer: dedaniel, robharad
---

# Get-SharedWithTeamUser

## SYNOPSIS
This cmdlet supports retrieving users of a shared with team.

## SYNTAX
```PowerShell
Get-SharedWithTeamUser -HostTeamId <String> -ChannelId <String> -SharedWithTeamId <String> [-Role <String>]
```

## DESCRIPTION
This cmdlet supports retrieving users of a shared with team.

## EXAMPLES

### Example 1
```PowerShell

> Get-AssociatedTeamUser -HostTeamId 126b90a5-e65a-4fef-98e3-d9b49f4acf12 -ChannelId 19:cUfyYYw3h_t-1KG8-WkvVa7KLEsIx-JHmyeG43VJojg1@thread.tacv2

Reviewer: ```suggestion
Get-SharedWithTeamUser -HostTeamId 126b90a5-e65a-4fef-98e3-d9b49f4acf12 -ChannelId 19:cUfyYYw3h_t-1KG8-WkvVa7KLEsIx-JHmyeG43VJojg1@thread.tacv2
```

-SharedWithTeam d2aad370-c6ca-438b-b4d7-05f0aa911a7b
```

> Returns users of a team with which specified channel is shared.

Reviewer 2: ```suggestion
Returns users of a team with which a specified channel is shared.
```

### Example 2
```PowerShell
Get-AssociatedTeam -HostTeamId 126b90a5-e65a-4fef-98e3-d9b49f4acf12 -ChannelId 19:cUfyYYw3h_t-1KG8-WkvVa7KLEsIx-JHmyeG43VJojg1@thread.tacv2
-SharedWithTeam d2aad370-c6ca-438b-b4d7-05f0aa911a7b -Role owner
```

> Returns owners of a team with which specified channel is shared.

Reviewer 2: ```suggestion
Returns owners of a team with which a specified channel is shared.
```

## PARAMETERS

### -HostTeamId

> Team id of the host team (group id)

Reviewer 2: ```suggestion
Team ID of the host team (group ID)
```

```yaml
Type: String
Parameter Sets: (All)
Aliases:

Required: True
Position: Named
Default value: None
Accept pipeline input: True (ByPropertyName)
Accept wildcard characters: False
```

### -ChannelId

> Thread id of the shared channel

Reviewer 2: ```suggestion
Thread ID of the shared channel.
```

```yaml
Type: String
Parameter Sets: (All)
Aliases:

Required: True
Position: Named
Default value: None
Accept pipeline input: True (ByPropertyName)
Accept wildcard characters: False
```

### -SharedWithTeamId

> Team id of the shared with team

Reviewer 2: ```suggestion
Team ID of the shared with team.
```

```yaml
Type: String
Parameter Sets: (All)
Aliases:

Required: True
Position: Named
Default value: None
Accept pipeline input: True (ByPropertyName)
Accept wildcard characters: False
```

### -Role

> Filter the results to only users with the given role: Owner or Member.

Reviewer 2: ```suggestion
Filters the results to only users with the given role of "Owner" or "Member".
```

-SharedWithTeam d2aad370-c6ca-438b-b4d7-05f0aa911a7b -Role owner
```

Returns owners of a team with which specified channel is shared.

## PARAMETERS

### -HostTeamId
Team ID of the host team (Group ID).

```yaml
Type: String
Parameter Sets: (All)
Aliases:

Required: True
Position: Named
Default value: None
Accept pipeline input: True (ByPropertyName)
Accept wildcard characters: False
```

### -ChannelId
Thread id of the shared channel

```yaml
Type: String
Parameter Sets: (All)
Aliases:

Required: True
Position: Named
Default value: None
Accept pipeline input: True (ByPropertyName)
Accept wildcard characters: False
```

### -SharedWithTeamId
Team ID of the shared with team.

```yaml
Type: String
Parameter Sets: (All)
Aliases:

Required: True
Position: Named
Default value: None
Accept pipeline input: True (ByPropertyName)
Accept wildcard characters: False
```

### -Role
Filters the results to only users with the given role of "Owner" or "Member".

```yaml
Type: String
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: True
Accept wildcard characters: False
```

### CommonParameters

> This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
> For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).

Reviewer 2: ```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```
